libdnf/sack/query.cpp

```diff
@@ -1058,6 +1059,30 @@ Query::Impl::filterPkg(const Filter & f, Map *m)
     map_init_clone(m, dnf_packageset_get_map(f.getMatches()[0].pset));
 }
 
+void
+Query::Impl::filterDepSolvable(const Filter & f, Map *m)
+{
+    assert(f.getMatchType() == _HY_PKG);
+    assert(f.getMatches().size() == 1);
+
+    dnf_sack_make_provides_ready(sack);
+    Pool *pool = dnf_sack_get_pool(sack);
```

Comment:
Please `Pool * pool`

Reply:
Also in all other new line please use a middle position for `*`.

Reply:
There were just a couple, I think I got them all.